sherpa/optmethods/optfcts.py

```diff
@@ -537,9 +537,9 @@ def stat_cb0( pars ):
 
     random.seed( seed  )
     if seed is None:
-        seed = random.randint(0, 2147483648L) # pow(2,31) == 2147483648L
+        seed = random.randint(0, 2147483648) # pow(2,31) == 2147483648L
     if population_size is None:
-        population_size = max( population_size, 12 * x.size )
+        population_size = 12 * x.size
```

Comment:
Why was the `max()` check removed here? Oh. hold on, population_size is by definition None here, so I see why you did it.

Comment left for anyone else who didn't spot this (ie me when I look at this again).